### PR TITLE
Add styling helper gems recipe

### LINKED ISSUE #3: Add styling helper gems recipe
We want the gems for:
- [Bourbon](https://github.com/thoughtbot/bourbon)
- [Bitters](https://github.com/thoughtbot/bitters)
- [Neat](https://github.com/thoughtbot/neat)

to have a recipe.

### PR DESCRIPTION
**Pull request by handerson**
Why?

> We want the gems for:
> - bourbon
> - bitters
> - neat
> 
> to have a recipe.
> closes https://github.com/spartansystems/spartan-composer-recipes/issues/3

How?
- add recipe that changes `app/assets/sytlesheets.css` to `.scss`
- add recipe for bourbon that includes the gem and imports bourbon
  into the scss file
- add recipe for bitters that includes the gem, runs the generator,
  and imports it into the scss file
- add recipe for neat that includes the gem

### CODE REVIEW

File: recipes/sass_setup.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+remove_file 'app/assets/stylesheets/application.css'
+create_file 'app/assets/stylesheets/application.scss'
```
Comment (Mrjaco12) [inline recipes/sass_setup.rb:2]: Do we want to keep any defaults in this file copied over from the one we're deleting?

Comment (handerson) [reply recipes/sass_setup.rb:2]: The default is to `require_tree .`, but it looks like we are favoring being explicit with our imports in our [raygun template](https://github.com/spartansystems/raygun-rails-template/blob/master/app/assets/stylesheets/application.scss).

Comment (Mrjaco12) [reply recipes/sass_setup.rb:2]: Cool. Makes sense.

**Review by Mrjaco12 — APPROVED** [review]

**Review by TheAznShumai — APPROVED** [review]

File: recipes/sass_setup.rb (hunk 1 above)
Comment (TheAznShumai) [reply recipes/sass_setup.rb:2]: 👍

**Pull request merged**